static/templates/stream_types.hbs

```diff
@@ -18,12 +19,14 @@
         </label>
     </li>
     {{#if is_admin}}
-    <li>
-        <label class="checkbox">
-            <input type="checkbox" name="is-announcement-only" value="is-announcement-only" {{#if is_announcement_only}}checked{{/if}}/>
-            <span></span>
-            {{t 'Restrict posting to organization administrators' }}
-        </label>
-    </li>
+        <h4>{{t 'Stream write settings'}}</h4>
```

Comment:
Make the first label "Who can access the stream?"

Maybe make this label "Who can post to the stream?"